.github/workflows/build_and_deploy.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           - host: windows-latest
             build: |
               npm i -g "@napi-rs/cli@${NAPI_CLI_VERSION}" "turbo@${TURBO_VERSION}" "pnpm@${PNPM_VERSION}"
-              turbo run build-native-no-plugin -- --release --target i686-pc-windows-msvc
+              turbo run build-native-no-plugin --summarize -- --release --target i686-pc-windows-msvc
```

Comment:
Your PR description says `--summary` but here you used `--summarize`.

Which one is correct?

Reply:
--summarize is correct, it's copied from existing ones (https://github.com/vercel/next.js/pull/51057/files#diff-8812abff9ed11e2e98d5066031a3181c6d44b28aea00f0d369532d5cb254650fR84). PR body is simple mistake.